site/content/docs/v1.5/upgrade-to-1.5.md

```diff
@@ -5,7 +5,7 @@ layout: docs
 
 ## Prerequisites
 
-- Velero [v1.4.x][5] installed.
+- Velero [v1.4.x][5], [v1.5.2][9], or [v1.5.1][8] installed.
```

Comment:
Really minor thing, but can we sort these in descending order as they are in the upgrade to 1.4 doc?